src/common/components/clusterWizard/clusterDetailsValidation.ts

```diff
@@ -1,6 +1,6 @@
 import * as Yup from 'yup';
 import { Cluster, ManagedDomain } from '../../api';
-import { OpenshiftVersionOptionType } from '../../types';
+import { CpuArchitecture, OpenshiftVersionOptionType } from '../../types';
```

Comment:
Looks like none of the changes in this file were necessary. Probably needs to be reverted.

Reply:
This errors will be caught by the new `'@typescript-eslint/no-unused-vars': 'error',` in Eslint.
@DanielleChason please rebase the branch against master and run the linter to fix them. Thanks!